lib/widget/image.dart

```diff
@@ -135,6 +139,20 @@ class ImageState extends WidgetState<EnsembleImage> {
     return rtn;
   }
 
+  Future<String> fetch(String url) async {
+    final http.Response response = await http
+        .get(Uri.parse("${url}timeStamp=${DateTime.now().toString()}"));
+    DateTime lastModifiedDateTime =
+        parseHttpDate("${response.headers['last-modified']}");
```

Comment:
what if the last-modified header is not present. You will end up sending "null" as the parameter. 

How will that work?